src/index.html

```diff
@@ -1,13 +1,377 @@
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="icon" href="./images/favicon.svg">
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__header header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a
+              href="/"
+              class="logo">
+              <img
+                class="logo__image"
+                src="./images/logo.svg"
+                alt="MyBiKE logo"
+              >
+            </a>
+
+            <div class="header__actions">
+              <a
+                href="tel:+1 234 5555-55-55"
+                class="icon icon--phone header__call"
+              ></a>
+
+              <a
+                href="#menu"
+                class="icon icon--menu"
+                ></a>
+            </div>
+          </div>
+
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a href="/" class="logo">
+              <img
+                class="logo__image"
+                src="./images/logo_dark.svg"
+                alt="MyBiKE logo">
+            </a>
+
+            <a href="#" class="icon icon--cross"></a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#compare-bikes" class="menu__link">Compare bikes</a>
+            </li>
+            <li class="menu__item">
+              <a href="#details" class="menu__link">Details</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call menu__call--phone"
+          >
+            +1 234 5555-55-55
+          </a>
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call menu__call--action"
+          >
+            Book a test ride
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-5">
+              <h2 class="about__title">Move Free</h2>
+            </div>
+
+            <div class="about__paragraph grid__item grid__item--7-11">
```

Comment:
i recommend separate classes on different lines (on whole page), but it up for you
```suggestion
            <div class="
              about__paragraph
              grid__item
              grid__item--7-11"
            >
``